src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java

```diff
@@ -68,14 +77,28 @@
  *   &lt;property name="illegalClassNames" value="NullPointerException"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *  public void func1() throws RuntimeException {} // ok
+ *  public void func2() throws NullPointerException {} // violation
```

Comment:
This should be indented with 2 spaces to match other examples.

Reply:
Done